jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -145,6 +149,42 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     return layers;
   }
 
+  /**
+   * Compute the entrypoint for a standard jar in exploded mode.
+   *
+   * @param jarPath path to jar file
+   * @param tempDirPath path to temporary jib local directory
+   * @param layers list of {@link FileEntriesLayer}
+   * @return list of {@link String} representing entrypoint
+   * @throws IOException if I/O error occurs when opening the jar file or if temporary directory
+   *     provided doesn't exist
+   * @throws IllegalArgumentException if main class is not found in the jar manifest
+   */
+  public static ImmutableList<String> computeEntrypoint_explodedStandard(
```

Comment:
Using `_` is only accepted in test method names. `computeEntrypointForExplodedStandard()`?

And I don't know who will require this method, but you should always double-checking if a method may have a narrower access scope. (Just don't habitually make everything `public` until necessary.)

Reply:
oops I keep mixing up the naming with python convention